eox_tagging/test/test_models.py

```diff
@@ -318,18 +318,49 @@ def test_create_without_default_owner(self):
                 "tag_type": "example_tag_3",
                 "validate_tag_value": {"regex": r".*eduNEXT$"},
                 "validate_target_object": "CourseEnrollment",
-                "validate_expiration_date": {"exist": True, "in": ["Dec 04 2020", "Oct 19 2020"]},
-                "validate_activation_date": "Jun 16 2020"
+                "validate_expiration_date": {"exist": True, "between": ["2020-10-19 10:20:30", "2020-12-04 10:20:30"]},
+                "validate_activation_date": "2020-06-16 10:20:30"
             }]
     )
     def test_validation_list_with_date(self):
-        """Used to test date validations."""
+        """
```

Comment:
could you verify that the tag was created in an assert section ? here the method doesn't fail however if someone add a broad exception in the method create_tag, this test will never fail